pkgs/by-name/gi/git-conventional-commits/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+let
+  version = "2.6.7";
+in
+buildNpmPackage {
+  pname = "git-conventional-commits";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "qoomon";
+    repo = "git-conventional-commits";
+    tag = "v${version}";
+    hash = "sha256-A5J5h6TrjPmMDQCTAn6TKTA71uu5Qe/Si32cx57vdmM=";
+  };
+
+  npmDepsHash = "sha256-8kf0Wx2YTjZDswFaLJzbxf4Tpjd3h47msOvU+JSBpWE=";
+
+  dontNpmBuild = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    homepage = "https://github.com/qoomon/git-conventional-commits";
+    description = "Generate semantic version, markdown changelogs, and validate commit messages";
```

Comment:
Nit: This isn't grammatically correct. Any ideas for a better description?

Reply:
How about changing **version** to the plural **versions** to align with the enumeration?

Reply:
SGTM